ui/src/main/webapp/app/components/app.component.ts

```diff
@@ -4,17 +4,17 @@ import {BreadCrumbsComponent} from './breadcrumbs.component';
 import {ApplicationListComponent} from './applicationlist.component';
 import {RouteConfig, RouterOutlet} from '@angular/router-deprecated';
 import {RegisterApplicationFormComponent} from "./registerapplicationform.component";
-import {MigrationProjectFormComponent} from "./MigrationProjectForm.component";
+import {MigrationProjectFormComponent} from "./migrationprojectform.component";
 
 @Component({
     selector: 'windup-app',
-    templateUrl: 'app/templates/app.component.html',
+    templateUrl: 'app/components/app.component.html',
```

Comment:
Yes, this is a step towards the guidelines in the Angular 2 style guide:
https://angular.io/styleguide#!#application-structure

The one directory per component approach may not be a bad thing to do either.